django/contrib/admin/static/admin/css/forms.css

```diff
@@ -261,7 +261,7 @@ body.popup .submit-row {
     overflow: auto;
 }
 
-.submit-row input {
+.submit-row input, .submit-row a.button {
```

Comment:
Why is this added?

Did you intend to remove `line-height` (and other) properties with the same values from later rules?

Reply:
You where right, it wasn't really needed... anyways I just refactored the css a little bit